### PR TITLE
Folijet fix for master

### PR DESCRIPTION
**Pull request by TetianaParanich**

### CODE REVIEW

File: cypress/e2e/data-import/delete-all-logs-from-landing-page.cy.js [hunk 1]
```diff
@@ -40,7 +40,9 @@ describe('ui-data-import: A user can delete import logs with "Data import: Can d
           const filePath = numberOfLogsToUpload - 1 === index ? filePathToUpload : emptyFilePathToUpload;
           fileNameToUpload = `C358137autotestFile.${getRandomPostfix()}.mrc`;
           DataImport.uploadFile(filePath, fileNameToUpload);
+          cy.wait(2000);
```
Comment (penfriend) [inline cypress/e2e/data-import/delete-all-logs-from-landing-page.cy.js:43]: please add the reason why you need this waiter

File: cypress/e2e/data-import/delete-all-logs-from-landing-page.cy.js [hunk 2]
```diff
@@ -40,7 +40,9 @@ describe('ui-data-import: A user can delete import logs with "Data import: Can d
...
           JobProfiles.searchJobProfileForImport('Default - Create instance and SRS MARC Bib');
+          cy.wait(5000);
```
Comment (penfriend) [inline cypress/e2e/data-import/delete-all-logs-from-landing-page.cy.js:45]: please add the reason why you need this waiter

**Review by penfriend — APPROVED** [review]

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/e2e/data-import/delete-all-logs-from-landing-page.cy.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/delete-all-logs-from-landing-page.cy.js:43]: I deleted it

File: cypress/e2e/data-import/delete-all-logs-from-landing-page.cy.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/e2e/data-import/delete-all-logs-from-landing-page.cy.js:45]: I deleted it

**Pull request merged**